### ISSUE TITLE
Major Bug: UpdateScanner 4.4.0b2 stopped displaying pages!

### ISSUE THREAD

**Issue by molitar** [sneakypete81/updatescanner#386]
To help explain the issue, please include the following:

    OS version: Windows 10 64 bit
    Firefox version: Firefox DE 65.02b
    Update Scanner version: 4.4.0 beta 2

Steps to reproduce the problem:
Unsure today I went to open my browser and opened up the Update Scanner sidebar. Clicked on one that said it was modified. I got a warning message for the first time ever saying it wanted permission to save file to my computer. I gave it permission and all I got was blank pages with the  UpdateScanner header and no Title link.  I tried to re-open several times by closing and restarting browser but it was totally broken.

Fix: Rolled back to UpdateScanner 4.4.0b1 and it's working great.

**Comment by yermulnik** [comment]
I'm having alike issue. Update Scanner version is 4.3.2. Example page is set to be scanned every 6 hours but the status shows `The page was last scanned 4 days ago.`

**Comment by yermulnik** [comment]
Seem like my issue got fixed by FF restart. At least manual scan started to work.

**Comment by sneakypete81** [comment]
@molitar: please start with a fresh profile and provide the exact steps to reproduce the problem:
https://github.com/sneakypete81/updatescanner/wiki#what-should-i-do-if-i-find-a-bug

**Comment by molitar** [comment]
Nothing more to report.  I woke up in the morning and opened up side bar and clicked on a bolded page that had update.  Nothing I had a blank page. I then tried several steps to recover.
 
1. Closed Firefox completely confirmed in taskmanager
2. re-opened same results
3. tried many of the pages I was monitoring
4. tried restarting computer
5. finally I tried going back to beta 1 version and it worked perfectly 
 
So it was not a lost of my configuration or profile as reinstalling the old version worked fine.  When it was showing no pages the header bar looked normal and even showed the time the page was updated but it would not load a page it was just blank as if it was not even trying to call the page open.

**Comment by molitar** [comment]
Well so far so good with updating again.. Strange why I would of had to downgrade then re-upgrade to make it work again.

**Comment by schmaudertamas** [comment]
A few days ago my sidebar is also empty.   
I checked my "Update Scanner's Pages" bookmark folder (FF 66.0.3) because I thought the folder will be empty and that is the source of the problem, but no. All the bookmarks are present in the folder. Than I created a new site to scan and checked where can i found this bookmark, (I thought something changed with the source of the sites to scan) but I couldnt find it :open_mouth:   
My updagte scanner version at the moment is 4.4.0 (updated today)

**Comment by molitar** [comment]
Yeah strange behavior anyone the way I got mine back was to install the previous version then re-update but Sneakypete may need some log or you to try some things so if you can hold off until he can respond if it happens again to me I plan to wait to see if there is something he wants me to test or try.

**Comment by btanner2** [comment]
Is this issue maybe related to:

https://github.com/sneakypete81/updatescanner/issues/453

**Comment by sneakypete81** [comment]
@schmaudertamas - your problem with the blank sidebar sounds more like #468. 
@molitar - have you seen this "warning message and blank pages" problem again? Are you able to reproduce it from a new profile?
